vcf2fhir/fhir_helper.py

```diff
@@ -443,6 +443,7 @@ def add_report_result(self):
         self.report.result = report_result
 
     def generate_final_json(self):
+        l1 = ["system", "code", "display"]

```

Comment:
If you want you can declare this at the top of the file as constant and rename as `CODE_ORDER`

Reply:
Please can you suggest a small name as CODE_ORDER is giving character limit error

Reply:
haha.. 🤦  `CG_ORDER` ? or `CG_ORD` of you are using `ORD` for ORDER do same thing `CODE_ORDER` as `CODE_ORD`